internal/worker/worker.go

```diff
@@ -152,8 +155,8 @@ func (w *BaseWorker) Start(ctx context.Context) error {
 	}
 }
 
-func (w *BaseWorker) executeCommandHandler(execCtx context.Context, errChan chan error, cmds []*cmd.DiceDBCmd, isWatchNotification bool) {
-	err := w.executeCommand(execCtx, cmds[0], isWatchNotification)
+func (w *BaseWorker) executeCommandHandler(execCtx context.Context, errChan chan error, cmds []*cmd.DiceDBCmd, isWatchChan bool) {
+	err := w.executeCommand(execCtx, cmds[0], isWatchChan)
```

Comment:
I feel this name is a little better?
```suggestion
	err := w.executeCommand(execCtx, cmds[0], isWatchNotification)
```